### PR TITLE
[modular]ish(?) adds more drip vendors and an oven to the ghost cafe

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

title

## Why It's Good For The Game

More drip is neato, and some of em have more than just drip.
Oven is needed for cooking stuff.

## Changelog
:cl:
expansion: added more drip vendors and an oven to the ghost cafe
/:cl:

### CODE REVIEW

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 1]
```diff
@@ -1975,13 +1975,10 @@
...
 	},
-/obj/machinery/vending/cigarette/syndicate{
-	custom_premium_price = 0;
-	custom_price = 0;
+/obj/machinery/vending/clothing{
 	default_price = 0;
 	extended_inventory = 1;
-	extra_price = 0;
-	fair_market_price = 0
+	vend_reply = "Thank! Enjoy your stay at the Cafe."
```
Comment (GoldenAlpharex) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm]:
```suggestion
	vend_reply = "Thanks! Enjoy your stay at the Cafe."
```

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 2]
```diff
@@ -5569,11 +5567,7 @@
 /turf/open/indestructible/hotelwood,
 /area/centcom/holding/cafepark)
 "aXd" = (
-/obj/machinery/vending/clothing{
-	default_price = 0;
-	extended_inventory = 1;
-	vend_reply = "Thank! Enjoy your stay at the Cafe."
-	},
+/obj/machinery/vending/access/command,
```
Comment (GoldenAlpharex) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm:5570]: Did you check that they actually have the access requited for this to work at all?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
In general, I'm not sure what the purpose of this is. They already get chameleon clothing, I fail to see how you would need anything else.

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm (hunk 2 above)
Comment (thestubborn) [reply _maps/map_files/generic/CentCom_skyrat_z2.dmm:5570]: i mean, they can access the other vendors just fine, I think their janitor pass is some sort of AA

**Comment by thestubborn** [comment]
> 
> 
> In general, I'm not sure what the purpose of this is. They already get chameleon clothing, I fail to see how you would need anything else.

the oven is actually needed, the rest...
well, considering I might add a firing range if this gets added, it'll be good to have armour
also, iirc you cant toggle chameleon stuff

**Comment by thestubborn** [comment]
> 
> 
> In general, I'm not sure what the purpose of this is. They already get chameleon clothing, I fail to see how you would need anything else.

image
it works

**Comment by tf-4** [comment]
> well, considering I might add a firing range if this gets added, it'll be good to have armour

No. We had it before. It was removed for a reason.

**Comment by thestubborn** [comment]
> 
> 
> > well, considering I might add a firing range if this gets added, it'll be good to have armour
> 
> No. We had it before. It was removed for a reason.

which was?
its not really good saying "it was removed for a reason" then not saying the reason mb
(most people use the cafe to fuck around and fight anyway)

**Review by GoldenAlpharex — COMMENTED** [review]
I mean, I'm just not sure why they really need those vendors considering that they have chameleon clothes anyway, and as stated, we're not turning the Ghost Cafe into a boxing ring. If you can give me a good reason for each of those vendors, sure, the oven is obviously very good, we have a ticket about it almost every round. I just want to understand your reasoning, now that we've cleared up that there won't be a firing range for the foreseeable future.

**Comment by thestubborn** [comment]
> 
> 
> I mean, I'm just not sure why they really need those vendors considering that they have chameleon clothes anyway, and as stated, we're not turning the Ghost Cafe into a boxing ring. If you can give me a good reason for each of those vendors, sure, the oven is obviously very good, we have a ticket about it almost every round. I just want to understand your reasoning, now that we've cleared up that there won't be a firing range for the foreseeable future.

I've not actually heard a reason beyond "It was removed for a reason.", which doesn't even tell me why
but again, chameleon clothing can't actually be toggled
i added them because there were other clothing vendors there, there should be consistency, either every vendor or none

**Comment by thestubborn** [comment]
image

**Pull request closed without merging**